The gear train in Figure Q1 is required to transmit the power of 40 kW at 1500 rpm from pinion 1. The ratio of speed between gear 1 and 2 is 5:2. Due to limited space, the ratio number of teeth on gear 3 to gear 4 is 1:3 with module of 4 mm and a pressure angle of 20. 

I) What does the 350 mm distance represent in the relationship between gear 1 and gear 2? 

II) What does the 200 mm distance represent in the relationship between gear 3 and gear 4? 

III) Find the number of teeth on gears 1, 2,3 and 4

IV) Find the radii of gears 1, 2,3 and 4

V)  Calculate the speed on gears 2, 3 and 4


In [6]:
from math import radians, pi, tan

Power = 40000
n1=1500
module=4
pressureAngle=radians(20)

$$\frac{n_1}{n_2}=\frac{5}{2}$$
$$\frac{N_3}{N_4}=\frac{1}{3}$$

I) What does the 350 mm distance represent in the relationship between gear 1 and gear 2? 

Center distance, c between Gear 1 and Gear 2

$$c=\frac{m}{2}(N_1+N_2)=350$$

II) What does the 200 mm distance represent in the relationship between gear 3 and gear 4? 

Center distance, c between Gear 3 and Gear 4

$$c=\frac{m}{2}(N_3+N_4)=200$$

III) Find the number of teeth on gears 1, 2,3 and 4

$$\frac{n_1}{n_2}=\frac{5}{2}=\frac{N_2}{N_1}$$
$$N_2=\frac{5}{2}N_1$$
$$\frac{m}{2}(N_1+N_2)=350$$

Substitute 1 to 2

$$\frac{m}{2}(N_1+\frac{5}{2}N_1)=350$$
$$\frac{4}{2}(N_1+\frac{5}{2}N_1)=350$$

In [2]:
N1=350/2*2/7
print(f'N1={N1}')
N2=5*N1/2
print(f'N2={N2}')

N1=50.0
N2=125.0


$$\frac{N_3}{N_4}=\frac{1}{3}$$
$$N_4=3N_3$$
$$\frac{m}{2}(N_3+N_4)=200$$

Substitute 3 to 4

$$\frac{m}{2}(N_3+N_4)=200$$
$$\frac{m}{2}(N_3+3N_3)=200$$

In [4]:
N3=200/2/4
print(f'N3={N3}')
N4=3*N3
print(f'N4={N4}')

N3=25.0
N4=75.0


IV) Find the radii of gears 1, 2, 3 and 4

Use module equation $m=\frac{d_1}{N_1}$

Gear 1

$$m=\frac{d_1}{N_1}$$
$$4=\frac{d_1}{50}$$
$$d_1=200mm$$
$$r_1=100mm$$

Gear 2

$$4=\frac{d_2}{125}$$
$$d_2=500mm$$
$$r_2=250mm$$

Gear 3

$$4=\frac{d_3}{25}$$
$$d_3=100mm$$
$$r_3=50mm$$

Gear 4

$$4=\frac{d_4}{75}$$
$$d_4=300mm$$
$$r_4=150mm$$

V)  Calculate the speed on gears 2, 3 and 4

From

$$\frac{n_1}{n_2}=\frac{5}{2}$$
$$\frac{1500}{n_2}=\frac{5}{2}$$
$$n_2=600rpm$$

Gear 2 and 3 mounted at the same shaft

$$n_3=n_2+600rpm$$

From

$$\frac{N_3}{N_4}=\frac{1}{3}$$
$$\frac{N_3}{N_4}=\frac{n_4}{n_3}$$
$$\frac{25}{75}=\frac{n_4}{600}$$
$$n_4=200rpm$$

6) FBD for Gear 1 and 2

7) Torque

$$P=\omega T$$
$$P=\frac{2\pi n_1}{60} T$$
$$40000=\frac{2\pi 1500}{60} T$$

In [4]:
Torque=round(Power*60/(2*pi*n1),2)
print(f'Torque={Torque} Nm')

Torque=254.65 Nm


8) Tangential and radial force

Use torque equation

$$T=F_{t1}\times r_1$$
$$254.65=F_{t1}(0.1)$$

In [5]:
Ft1 = Torque/0.1
print(f'Ft1={Ft1}N')

Ft1=2546.5N


Radial force

$$F_{r1}=F_{t1} \tan(20)$$
$$=(2546.5)\tan(20)$$

In [9]:
Fr1=round(Ft1*tan(pressureAngle),2)
print(f'Fr1={Fr1}N')

Fr1=926.85N


$$F_{t1}=F_{t2}$$
$$F_{r1}=F_{r2}$$

9) Power transmitted across gears are ideally same if no losses

10) Torque are different if diameter is different

11) FBD between Gear 3 and 4

12) Torque for Gear 3

From

$$P_3=\frac{2\pi n_3}{60}T_3$$
$$40000=\frac{2\pi 600}{60}T_3$$

In [11]:
Torque3=round(Power*60/(2*pi*600),2)
print(f'Torque at Gear 3, T3={Torque3}Nm')

Torque at Gear 3, T3=636.62Nm


13) Use torque equation

$$T=F_{t3}(0.05)$$

In [12]:
Ft3=Torque3/0.05
print(f'Ft3={Ft3}Nm')

Ft3=12732.4Nm


$$F_{r3}=F_{t3} \tan(20)$$

In [14]:
Fr3=round(Ft3*tan(pressureAngle),2)
print(f'Fr3={Fr3}N')

Fr3=4634.21N


$$F_{t3}=F_{t4}$$
$$F_{r3}=F_{r4}$$